smart_open/smart_open_lib.py

```diff
@@ -573,23 +588,6 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         pass
 
-
-def make_closing(base, **attrs):
```

Comment:
Oops, sorry, didn't think those comments were still active since python 2.7 has been around so long and there's no reason I know of for people to stick with 2.6. I can revert this part if you need 2.6 support -- let me know.